drivers/driver_manager.ts

```diff
@@ -1,11 +1,12 @@
-import { FileSystemDriver } from './drivers.js'
-import fileSystemDriveBuilder from './file_system_driver.js'
+import FileSystemDriver from "./file_system_driver.js"
```

Comment:
.js ??

Reply:
yes, we're using `type=module` in package.json